.travis.yml

```diff
@@ -48,7 +45,6 @@ script:
   - go build -tags 'coprocess python'
   - go build -tags 'coprocess lua'
   - go build -tags 'coprocess grpc'
-  - ./bin/ci-swagger.sh
```

Comment:
We still need to validate the swagger file is valid at all times. Not sure these lines should go off,